packages/core/ai_content_pipeline/ai_content_pipeline/cli/commands/motion.py

```
@@ -0,0 +1,50 @@
+@click.option("-i", "--image", required=True, help="Image file path or URL (character/background source)")
+@click.option("-v", "--video", required=True, help="Video file path or URL (motion source)")
+@click.option("-o", "--output", "output_dir", default="output", help="Output directory")
+@click.option("--orientation", default="video",
+              type=click.Choice(["video", "image"]),
+              help="Character orientation: video (max 30s) or image (max 10s)")
+@click.option("--no-sound", is_flag=True, default=False,
+              help="Remove audio from output (default: keep sound)")
+@click.option("-p", "--prompt", default=None, help="Optional text description to guide generation")
+@click.option("--save-json", default=None, help="Save result metadata as JSON file")
+@click.pass_context
+def transfer_motion_cmd(ctx, image, video, output_dir, orientation, no_sound, prompt, save_json):
```

Ruff (0.14.14)

[warning] Unused function argument: `ctx`

(ARG001)